src/data/loadToDb.ts

```diff
@@ -0,0 +1,44 @@
+//import AWS from 'aws-sdk';
+import { productsDB } from './productsForLoad';
+import { stockDB } from './stockForLoad';
+import { IProduct, IStock, StatusCode } from '../types/types';
+import { generateHttpResponse } from '../utils/lambda';
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, PutCommand } from "@aws-sdk/lib-dynamodb";
+
+const dynClient = DynamoDBDocumentClient.from(
+    new DynamoDBClient({ region: "eu-west-1" })
```

Comment:
Ideally we should not hardcode region, but pass it via environment variables from serveless.ts file

Reply:
Done